_includes/cockroachcloud/create-a-sql-user.md

```diff
@@ -0,0 +1,17 @@
+{% include cockroachcloud/cockroachcloud-ask-admin.md %}
+
+1. Navigate to your cluster's **SQL Users** page.
+
+    <img src="{{ 'images/v19.2/cockroachcloud/sql-users.png' | relative_url }}" alt="SQL users" style="border:1px solid #eee;max-width:100%" />
+
+2. Click the **Add User** button in the top right corner.
+
+    The **Add User** modal displays.
+
+3. Enter a **Username** and **Password**.
+    {{site.data.alerts.callout_info}}
+    Password must be at least 12 characters long.
```

Comment:
This feels like something we should leave to UI to validate (also, it will probably change)